examples/audio/maxmod/audio_modes/source/main.c

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identiier: CC0-1.0
+//
+// SPDX-FileContributor: Michele Di Giorgio, 2024
+
+// Example usage of the following audio modes:
+// - MM_MODE_A, Complete hardware mixing mode
+// - MM_MODE_B, Interpolated mixing mode
+// - MM_MODE_C, Extended mixing mode
+//
+// Check https://maxmod.org/ref/functions/mmSelectMode.html for more
```

Comment:
Eh... that's the devkitPro-maintained documentation 😅 You can link here instead: https://blocksds.github.io/docs/maxmod/group__nds__arm9__init.html